probe-rs/src/bin/probe-rs/cmd/read.rs

```diff
@@ -0,0 +1,58 @@
+use probe_rs::MemoryInterface;
+
+use crate::util::common_options::ProbeOptions;
+use crate::util::common_options::ReadWriteBitWidth;
+use crate::util::common_options::ReadWriteOptions;
+use crate::CoreOptions;
+
+#[derive(clap::Parser)]
+pub struct Cmd {
```

Comment:
Could we add a helptext to this command, on how the expected output will look like (e.g. hex numbers little endian) with some example as well? :)